cutereport/cutereport.2016-01-17.manifest

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://inqlude.org/schema/release-manifest-v1#",
+  "name": "cutereport",
+  "display_name": "CuteReport",
+  "version": "1.1.1",
```

Comment:
This is the same version numbers as in the first file. Is this supposed to be 1.2?